diff --git a/packages/react-native/lib/commonjs/generated/breez_sdk_liquid.js b/packages/react-native/lib/commonjs/generated/breez_sdk_liquid.js
index a6120880..b21a3a9c 100644
--- a/packages/react-native/lib/commonjs/generated/breez_sdk_liquid.js
+++ b/packages/react-native/lib/commonjs/generated/breez_sdk_liquid.js
@@ -38,9 +38,12 @@ function connectWithSigner(req, signer) /*throws*/{
   }, /*liftString:*/FfiConverterString.lift));
 }
 function defaultConfig(network, breezApiKey) /*throws*/{
-  return FfiConverterTypeConfig.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypeSdkError.lift.bind(FfiConverterTypeSdkError), /*caller:*/callStatus => {
+  const config = FfiConverterTypeConfig.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypeSdkError.lift.bind(FfiConverterTypeSdkError), /*caller:*/callStatus => {
     return (0, _breez_sdk_liquidFfi.default)().ubrn_uniffi_breez_sdk_liquid_bindings_fn_func_default_config(FfiConverterTypeLiquidNetwork.lower(network), FfiConverterOptionalString.lower(breezApiKey), callStatus);
   }, /*liftString:*/FfiConverterString.lift));
+  var fs = require('react-native-fs');
+  config.workingDir = `${fs.DocumentDirectoryPath}/breezSdkLiquid`;
+  return config;
 }
 function parseInvoice(input) /*throws*/{
   return FfiConverterTypeLNInvoice.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypePaymentError.lift.bind(FfiConverterTypePaymentError), /*caller:*/callStatus => {
diff --git a/packages/react-native/lib/module/generated/breez_sdk_liquid.js b/packages/react-native/lib/module/generated/breez_sdk_liquid.js
index bf7db559..f85a715b 100644
--- a/packages/react-native/lib/module/generated/breez_sdk_liquid.js
+++ b/packages/react-native/lib/module/generated/breez_sdk_liquid.js
@@ -25,9 +25,12 @@ export function connectWithSigner(req, signer) /*throws*/{
   }, /*liftString:*/FfiConverterString.lift));
 }
 export function defaultConfig(network, breezApiKey) /*throws*/{
-  return FfiConverterTypeConfig.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypeSdkError.lift.bind(FfiConverterTypeSdkError), /*caller:*/callStatus => {
+  const config = FfiConverterTypeConfig.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypeSdkError.lift.bind(FfiConverterTypeSdkError), /*caller:*/callStatus => {
     return nativeModule().ubrn_uniffi_breez_sdk_liquid_bindings_fn_func_default_config(FfiConverterTypeLiquidNetwork.lower(network), FfiConverterOptionalString.lower(breezApiKey), callStatus);
   }, /*liftString:*/FfiConverterString.lift));
+  var fs = require('react-native-fs');
+  config.workingDir = `${fs.DocumentDirectoryPath}/breezSdkLiquid`;
+  return config;
 }
 export function parseInvoice(input) /*throws*/{
   return FfiConverterTypeLNInvoice.lift(uniffiCaller.rustCallWithError(/*liftError:*/FfiConverterTypePaymentError.lift.bind(FfiConverterTypePaymentError), /*caller:*/callStatus => {
@@ -10855,4 +10858,4 @@ export default Object.freeze({
     FfiConverterTypeWalletInfo
   }
 });
